packages/react-sdk/src/components/Connect.tsx

```diff
@@ -11,32 +17,98 @@ import {StyleWrapper} from "./internal/StyleWrapper"
 import {UserIcon} from "../icons/UserIcon"
 import {CopyIcon} from "../icons/CopyIcon"
 import {LogOutIcon} from "../icons/LogOutIcon"
+import {ExternalLinkIcon} from "../icons/ExternalLink"
 
 type BalanceType = keyof UseCrossVmTokenBalanceData
 
+export interface TokenConfig {
+  symbol: string
+  name: string
+  vaultIdentifier?: string
+  erc20Address?: string
+}
```

Comment:
I don't think symbol is needed here, it should be implicit (optional override is OK)

Reply:
What do you mean by implicit? To me having both is useful because the symbol it is a direct property of a token and the actual ticker, while the name is more descriptive friendly label. An example is:

**stFlow (Increment Staked FLOW)**
symbol: stFlow
name: stFlow (Increment Staked FLOW)

or

**USDF (PYUSD)**
symbol: USDF
name: USDF (PYUSD)